src/index.html

```diff
@@ -1,12 +1,400 @@
+<!-- #region head -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <div class="header__nav">
+        <a href="#home" class="header__logo">
+          <img src="./images/logo.png" alt="MOYO">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">APPLE</a>
+            </li>
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">SAMSUNG</a>
+            </li>
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">SMARTPHONES</a>
+            </li>
+            <li class="nav__item">
+              <a href="#laptops-computers" data-qa="nav-hover"
+              class="nav__link">LAPTOPS & COMPUTERS</a>
```

Comment:
Fix with code style and check it everywhere
```suggestion
              <a
                href="#laptops-computers"
                data-qa="nav-hover"
                class="nav__link"
              >
                LAPTOPS & COMPUTERS
              </a>
```